tests/integration/targets/nxos_lag_interfaces/tests/common/parsed.yaml

```diff
@@ -14,6 +14,7 @@
           interface port-channel12
           interface Ethernet1/800
             channel-group 10 mode active
+            no shutdown
```

Comment:
From the issue #147 , I see that, mode is set to "no" only for interfaces, with 

`interface Ethernet1/50`
`..............`
`channel-group 5 ----------------------------> mode = on`
`no shutdown
`

When the configuration is **channel-group 100 mode active** , the "mode" key in gathered facts is set to "active" as expected.

So "no shutdown" can be added below a channel-group configuration with mode set to "on" ( no mode in the config )

Reply:
You're right. Fixed it.